src/app/teams/teams.component.ts

```diff
@@ -106,7 +109,8 @@ export class TeamsComponent implements OnInit, AfterViewInit {
     });
   }
 
-  addTeam(team?) {
+  addTeam(team?, e) {
```

Comment:
This is causing a build error because required function parameters must come before optional ones.